roles/nginxplus/templates/logrotate.d.app_protect.j2

```diff
@@ -0,0 +1,12 @@
+# logrotate configuration for app_protect in {{ ansible_managed | comment }}
+/var/log/app_protect/*.log {
```

Comment:
Do the DoS protection logs write to the same directory? If so, we'd be prepared for the change-over when our license renews (and also for dev).

Reply:
DOS does not write security logs.